### ISSUE TITLE
[Package Issue]: Google.PlayGames.Beta

### ISSUE THREAD

**Issue by angelul** [microsoft/winget-pkgs#154041]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

No applicable update found.

### Brief description of your issue

The PR #153924 merged a manifest with an installer path that downloads the Google Chrome installer instead of the Play Games installer, also, maybe there is no new version of Play Games, only there are for Google Chrome.

### Steps to reproduce

Install Google.PlayGames.Beta with 24.4.932.3 version.

Exec `winget upgrade --id Google.PlayGames.Beta`

### Actual behavior

It will try to upgrade to 126.0.6462.0, but instead of this, it will download and exec the Google Chrome installer

### Expected behavior

Download and exec the Google Play Games installer

### Environment

```raw
Administrador de paquetes de Windows v1.7.11261
Copyright (c) Microsoft Corporation. Todos los derechos reservados.

Windows: Windows.Desktop v10.0.22631.3593
Arquitectura del sistema: X64
Paquete: Microsoft.DesktopAppInstaller v1.22.11261.0

Directorios de WinGet
-----------------------------------------------------------------------------------------------------------------------
Registros                                   %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalS…
Configuración de usuario                    %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalS…
Directorio de vínculos portátiles (usuario) %LOCALAPPDATA%\Microsoft\WinGet\Links
Directorio de vínculos portátiles (equipo)  C:\Program Files\WinGet\Links
Raíz de paquete portátil (usuario)          %LOCALAPPDATA%\Microsoft\WinGet\Packages
Raíz de paquete portátil                    C:\Program Files\WinGet\Packages
Raíz de paquete portátil (x86)              C:\Program Files (x86)\WinGet\Packages
Descargas del instalador                    %USERPROFILE%\Downloads

Vínculos
-----------------------------------------------------------------------------------
Declaración de privacidad   https://aka.ms/winget-privacy
Contrato de licencia        https://aka.ms/winget-license
Avisos de terceros          https://aka.ms/winget-3rdPartyNotice
Página principal            https://aka.ms/winget
Términos de Microsoft Store https://www.microsoft.com/en-us/storedocs/terms-of-sale

Configuración de administrador            Estado
-------------------------------------------------------
LocalManifestFiles                        Deshabilitado
BypassCertificatePinningForMicrosoftStore Deshabilitado
InstallerHashOverride                     Deshabilitado
LocalArchiveMalwareScanOverride           Deshabilitado
```

**Comment by CanePlayz** [comment]
This will be fixed with #154053

**Comment by aisgbnok** [comment]
Close with reason: Resolved by above linked PR;

**Comment by stephengillie** [comment]
> @angelul, seems like this issue has been resolved by #154053.
> 
> If that is the case this issue can be closed.
